src/Network/Skylark/Core/Types.hs

```diff
@@ -217,85 +214,86 @@ instance Var LogLevel where
 
 instance (Var a, Show a, Read a) => Var (Maybe a) where
   toVar (Just v) = (unpack . show) v
-  toVar Nothing = ""
+  toVar Nothing  = ""
+
   fromVar = readMaybe
 
 -- | A record type representing full or partial configuration of an
 -- HTTP service. Remaining unspecified fields are filled in with the
 -- default values from Default.
 --
-data Config = Config
-  { _cConfigFile    :: Maybe String   -- ^ Service configuration file location
```

Comment:
I think the `_c` prefix is going to be annoying here in causing conflicts - for these base types that get extended, I've been going for short abbreviated names where possible (`Ctx`, `Env`, etc.), and using the entire name spelled out as a prefix - e.g., `_ctx` and `_env`.